### PR TITLE
Fix nav tabs bug and change a word on eventregistration.json

### PR DESCRIPTION
**Pull request by brifemu**

### CODE REVIEW

File: src/assets/js/eventregistration.js [hunk 1]
```diff
@@ -3,8 +3,13 @@ import { proto } from './lib/trace_location_pb';
 import { encode } from 'uint8-to-base64';
 import Cleave from 'cleave.js';
 import Papa from 'papaparse';
+import $ from 'jquery';
 import 'regenerator-runtime/runtime'
```
Comment (svengabr) [inline src/assets/js/eventregistration.js:8]: Please use the existing tab javascript:
https://github.com/corona-warn-app/cwa-website/blob/master/src/assets/js/app.js#L233-L241

**Review by svengabr — CHANGES_REQUESTED** [review]
Waiting for javascript to be resolved

File: src/assets/js/eventregistration.js (hunk 1 above)
Comment (brifemu) [reply src/assets/js/eventregistration.js:7]: Ok, done as you suggested

**Comment by brifemu** [comment]
HTML tabs refactored taking by reference the tabs coded on the site.

**Pull request merged**